doc/Overview/Reactive/overview.md

```diff
@@ -0,0 +1,148 @@
+﻿---
+uid: Overview.Reactive.Feed
+---
+
+# MVUx Overview
+
+## What is MVU
+
+MVU stands for **M**odel-**V**iew-**U**pdate.  
+
+It utilizes a flow where the **model** is rendered on the **view** side, and once changed by request from user, gets **update**d.
+
+It enables and promotes leaving the actual _model_ plain and immutable,
+making the whole need of raising property change notification redundant.
```

Comment:
Please leverage chatgpt or bing chat for a clearer starting point for definition of MVU:
![Uploading image.png…]()